source/kafka-connector-atlas-stream-processing-comparison.txt

```diff
@@ -0,0 +1,83 @@
+.. _kafka-connector-atlas-stream-processing-comparison:
+
+===================================================
+Compare Kafka Connector and Atlas Stream Processing
+===================================================
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 2
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page provides a comparison of the feature sets of the MongoDB
+Kafka Connector and {+atlas-sp+} to help you identify which tool
+best suits your use case.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40 40
+
+   * - Feature
+     - Kafka Connector
+     - Atlas Stream Processing
```

Comment:
General comment: Which of these two solutions we want to promote? If asp, then should it be first?